ci-operator/templates/openshift/openshift-azure/rbac.yaml

```diff
@@ -0,0 +1,26 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: NAMESPACE
+  required: true
+
+objects:
+
+# We want the cluster to be able to access these images
```

Comment:
This rolebinding allows the cluster to pull images as `unauth` and `auth`.  Is this here to allow us to pull from other namespaces?

Reply:
This is "hack". By default ci-operator builds all containers into `stable` ImageStream. And those are accessible only for namespace process. We use template functionality to open them up, so our cluster in azure could access them and pull "test images". So we "opens our test namespace to the world"